youtube_dl/downloader/f4m.py

```diff
@@ -331,6 +331,21 @@ def frag_progress_hook(status):
             eta = self.calc_eta(start, time.time(), estimated_size, byte_counter)
             self.report_progress(progress, format_bytes(estimated_size),
                                  status.get('speed'), eta)
+
+            # report totals to actual hooks
+            toplevel_status = {'status': 'downloading', 'filename': filename, 'tmpfilename': tmpfilename}
+            toplevel_status['downloaded_bytes'] = state['downloaded_bytes']
+            total_bytes = 0
+            if progress > 0:
```

Comment:
`progress` is a value for human consumption. I'm pretty sure you meant `state['downloaded_bytes']` here.